AlgorithmiaTests/AlgorithmiaTests.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  AlgorithmiaTests.swift
+//  AlgorithmiaTests
+//
+//  Created by Erik Ilyin on 11/9/16.
+//  Copyright © 2016 algorithmia. All rights reserved.
+//
+
+import XCTest
+@testable import Algorithmia
+
+class AlgorithmiaTests: XCTestCase {
+    var client:AlgorithmiaClient?
+    override func setUp() {
+        super.setUp()
+        
+        let ALGORITHMIA_API_KEY = "" // Your API Key
```

Comment:
Modifying code to run tests seems like bad style. Can we use an environment variable with the same name?